### PR TITLE
Update tests

### PR DESCRIPTION
**Pull request by Cynerva**
First pass at pulling in tests from bundle-canonical-kubernetes. I have not run these yet, working on that next.

See https://github.com/juju-solutions/bundle-kubernetes-core/commit/ad066069f343bd76494f25e5ca62218ac385a2c5 for the changes I made to the tests. Everything else looked reasonable to me, but it's possible I missed something.

@mbruzek @chuckbutler @wwwtyro

### CODE REVIEW

File: tests/tests.yaml [hunk 1]
```diff
@@ -1,4 +1,10 @@
+# Bundle level test. Only execute the 20 and 30 class series of tests
```
Comment (lazypower) [inline tests/tests.yaml]: This comment can go. It's stale info for a failed concept

File: tests/tests.yaml [hunk 2]
```diff
@@ -1,4 +1,10 @@
...
+tests: "*"
+# Adding tims PPA so we get current dependencies
+sources:
+  - ppa:tvansteenburgh/ppa
```
Comment (lazypower) [inline tests/tests.yaml:4]: If the tyou st is executed in charmbox this won't be necessary. Not a nack but just FYI

**Comment by lazypower** [comment]
Two minor comments left inline. This lgtm if we get greens on prelim testing

**Review by lazypower — COMMENTED** [review]
See comment in conversation view

File: tests/tests.yaml (hunk 2 above)
Comment (lazypower) [reply tests/tests.yaml:4]: *if the test

Stupid auto fail on my phone keyboard

**Comment by mbruzek** [comment]
I have tested this bundle and got all the tests to pass.  +1 from me too. Thanks!

```
cs:~containers/flannel-4
    charm-proof                                                            PASS
    make lint                                                              PASS
cs:~containers/etcd-14
    charm-proof                                                            PASS
    make lint                                                              PASS
    10-deploy.py                                                           PASS
cs:~containers/kubernetes-worker-5
    charm-proof                                                            PASS
    make lint                                                              PASS
cs:~containers/easyrsa-2
    charm-proof                                                            PASS
    make lint                                                              PASS
cs:~containers/kubernetes-master-4
    charm-proof                                                            PASS
    make lint                                                              PASS
bundle
    charm-proof                                                            PASS
    20-basic-component-check.py                                            PASS

PASS: 13 Total: 13 (424.00962 sec)
```

**Pull request merged**